data/observational/orm/platform.py

```diff
@@ -0,0 +1,51 @@
+from types import MappingProxyType
+from typing import Dict, List
+import enum
+
+from data.observational import db
+
+class Platform(db.Model):
+    __tablename__ = 'platforms'
+
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
For anyone else down the road we don't need to explicitly set `auto_increment` here since sqlalchemy sets the first non-foreign-key integer column to auto increment.